roles/add-vhost/tasks/main.yml

```diff
@@ -57,19 +57,3 @@
     register: inventory_hostname_domain_response
   when:
     inventory_hostname != acsf_site_name
-
-# Ex. sunetid.people.stanford.edu
-- name: Add SUNetID as custom domain
```

Comment:
This removal is tangential to the substance of this PR. I just noticed that we don't need it anymore now that we have the Customer Standard Domain of people.stanford.edu for the leland stack.

Reply:
Fine by me.